src/main/java/me/csed2/moneymanager/subscriptions/commands/UpdateSubscriptionCommand.java

```diff
@@ -0,0 +1,62 @@
+package me.csed2.moneymanager.subscriptions.commands;
+
+import me.csed2.moneymanager.categories.CategoryCache;
+import me.csed2.moneymanager.command.ICommand;
+import me.csed2.moneymanager.subscriptions.Subscription;
+import me.csed2.moneymanager.subscriptions.SubscriptionArgType;
+import me.csed2.moneymanager.subscriptions.SubscriptionCache;
+
+public class UpdateSubscriptionCommand<T> implements ICommand<Boolean> {
```

Comment:
Maybe add an ability to update the time cycle here?

Reply:
> Maybe add an ability to update the time cycle here?

Yeah, I will do I just wanted to push the changes I made to the file structure to avoid any major conflicts later on.